4term/nodejs/lab5/package.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "lab5",
+    "version": "1.0.0",
+    "main": "src/app.ts",
```

Comment:
Add "type": "module" to your package.json.
https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c